### ISSUE TITLE
Bug: Task tray icon does not appear in Windows 10 environment

### ISSUE THREAD

**Issue by hijiro-gthb** [files-community/Files#14610]
### Description

The task tray icon is displayed in the Windows 11 environment, but not in the Windows 10 environment.

Note that the "Keep apps running in the background when windows are closed" option is turned on, and the Files process itself seems to be resident when checked in Task Manager.

![Files - Task tray icon does not appear in Windows 10 environment](https://github.com/files-community/Files/assets/157790562/b1e5b844-42eb-4481-a7f9-aab6e9b235a6)

### Steps To Reproduce

It does not appear with Files activated, nor does it appear when closed with the close button.

### Requirements

Windows 10 environment.

### Files Version

3.1.10.0

### Windows Version

10.0.19045.3996

### Log File

[debug.log](https://github.com/files-community/Files/files/14122235/debug.log)

**Comment by Qin-shihuang** [comment]
I can reproduce this issue. On my local machine, the Publish and Preview versions do not have tray icons. The Dev version has an icon, but the icon also disappears when the window is closed. The program still runs in the background after the window is closed.

**Comment by Josh65-2201** [comment]
Thanks for the report, Can you make sure there enabled in Windows taskbar settings and there not in the overflow menu.

**Comment by hijiro-gthb** [comment]
Sorry.
Perhaps because I am using the Japanese version of Windows, the proper nouns do not seem to be the same, and perhaps my answer is off.

In the taskbar settings of Windows 10, you can control the following toggling, right?
- Icon that is always displayed in the notification area
- Icons that are displayed by expanding from the arrow on the left side of the notification area

In this setting screen, the Files item does not exist.
Does this answer your question?

![Files - taskbar settings of Windows 10](https://github.com/files-community/Files/assets/157790562/e19df8a0-5b11-4457-8a21-b92e10313179)

**Comment by yaira2** [comment]
Can you try fully closing Files by running this cmd `taskkill -f -im files.exe` and starting the app again?

**Comment by hijiro-gthb** [comment]
I have tried killing and restarting the Files process as you suggested, but no change.

As I wrote previously, when I close the Files window, the following state occurs.
- The task tray icon of Files does not exist.
- The background process of Files is resident.

Sorry if I am wrong.
To begin with, I don't think this will solve the problem, since kill & restart will only cause the same situation as at Windows bootup.

Incidentally, when I perform the kill with the command you provided, Files.exe and Files.App.Server.exe die, but the Runtime Broker.exe that is tied to them seems to remain.
(See attached image)

![Files - Runtime Broker in Task Manager](https://github.com/files-community/Files/assets/157790562/f5bd26a9-b476-417b-a4eb-fa7a3e50dcfb)

**Comment by yaira2** [comment]
Do you by any chance have both the stable and preview version set to open at Windows Startup?

**Comment by yaira2** [comment]
@hijiro-gthb

**Comment by hijiro-gthb** [comment]
I'm sorry.
My reply was delayed due to health reasons.

I have only installed and used the preview version.

**Comment by yaira2** [comment]
> Can you try fully closing Files by running this cmd `taskkill -f -im files.exe` and starting the app again?

Did running this command fully close the process?

**Comment by hijiro-gthb** [comment]
Sorry if I have been misguided.

I've sorted out the situation once again.
(Sorry for the length of this post.)

----------------
1. "Files" app is in the foreground
Looking in Task Manager, there is a "Files - Preview" process of type "Apps".
**The task tray icon for the "Files" app does not exist.**
![Files - Task Tray icon not found_20240220_01](https://github.com/files-community/Files/assets/157790562/766436b5-5748-4a83-b022-1772244c37e5)
----------------
2. "Files" application is closed by pressing the "x" button
Task Manager shows that the "Files - Preview" process of type "Background Process" exists.
**The task tray icon for the "Files" app does not exist.**
![Files - Task Tray icon not found_20240220_02](https://github.com/files-community/Files/assets/157790562/1d2fda90-64c8-4d7a-957d-c89d98e110d5)
----------------
3. "Files" application is completely terminated by "taskkill -f -im files.exe" command
For a while, only the "Runtime Broker" associated with "Files - Preview" remained, but it disappeared after about 20 seconds.
_**I said before that the "Runtime Broker" did not disappear, but I will partially correct that.**_
On the Task Manager, the "Files - Preview" process does not exist, **and of course the task tray icon of the "Files" app does not exist either.**
----------------
4. The "Files" app was launched again and placed in the foreground
Since the process was completely terminated in step 3, it takes a long time to start up. (I think this is the correct behavior)
Looking at the Task Manager, the "Files - Preview" process of type "App" is present, but unlike in 1., the "Runtime Broker" is not present.
**The task tray icon for the "Files" app does not exist.**
![Files - Task Tray icon not found_20240220_04](https://github.com/files-community/Files/assets/157790562/c0bbc06d-18cf-447f-ac91-87cdfe8bec0d)
----------------
5. The application is closed with the "x" button of the "Files" application again
Looking at the Task Manager, the "Files - Preview" process of type "Background Process" exists, but the "Runtime Broker" does not exist as in 4.
**The task tray icon for the "Files" app is not present.**
![Files - Task Tray icon not found_20240220_05](https://github.com/files-community/Files/assets/157790562/3c25a5da-a09e-4a79-9300-d09a5a86e8da)

**Comment by yaira2** [comment]
Is this still an issue in the preview version? https://files.community/download/preview

**Comment by Josh65-2201** [comment]
@hijiro-gthb Is this still an issue on v4.0.11 https://files.community/download

**Comment by hijiro-gthb** [comment]
Yes, the taskbar icon remains hidden in the following environment:

Windows 10 Pro 22H2 (Build 19045.6390)
Files 4.0.11

**Comment by hijiro-gthb** [comment]
**Correction**: It is the “task tray icon,” not the “taskbar icon.”

**Comment by Josh65-2201** [comment]
Thanks for checking, is it the same Windows install as before?

**Comment by hijiro-gthb** [comment]
Yes, this bug has persisted on the device where it was first identified.

**[Additional Notes]**
Of course, the Windows 10 on that device is always kept up to date, so the build number has changed.
However, Windows itself has not been reinstalled.

**Comment by Josh65-2201** [comment]
Thanks, this seem like a Windows issue as a fresh install of 22h2 (build 19045) show it correctly for me
